### ISSUE TITLE
calendar is not allowing to re-select a date

### ISSUE THREAD

**Issue by IngenieroLata** [primefaces/primeng#2261]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Current behavior**

The `calendar` component is not allowing to re-select a deleted date 
![calendar](https://cloud.githubusercontent.com/assets/5667514/23758585/40931b02-04b8-11e7-9854-a25c1567d86b.gif)

**Expected behavior**

When deleting a selected date, the exact same date should be abailable to select it again and show it on the input field

**Minimal reproduction of the problem with instructions**

- Click in a calendar component 
- Choose a date (could be with hours and minutes)
- Delete the date that appears in the input field
- Try to select the exact same date again

**What is the motivation / use case for changing the behavior?**

Trying to update a prefilled form

**Please tell us about your environment:**

* **Angular version:** 2.4.3

* **PrimeNG version:** 1.1.4

* **Browser:** Chrome 56.0.2924.87 (64-bit) 

 
* **Language:** TypeScript 2.1.5

* **Node (for AoT issues):** v6.9.1

**Comment by cagataycivici** [comment]
Can you try with 2.0.2 please.

**Comment by IngenieroLata** [comment]
@cagataycivici yes, I did it just for test and actually I tried algo in the primeng official page and got the same behavior.

**Comment by external-olgierd** [comment]
Seems like the same issue as #1594.

**Comment by IngenieroLata** [comment]
So this is being closed and continue in #1594 ?

**Comment by cagataycivici** [comment]
Fixed via 1594
